feature-libs/organization/administration/components/unit/details/unit-details.component.html

```diff
@@ -14,7 +14,12 @@
     i18nRoot="orgUnit"
   ></cx-org-toggle-status>
 
-  <cx-org-disable-info info i18nRoot="orgUnit"> </cx-org-disable-info>
+  <cx-org-disable-info
+    info
+    i18nRoot="orgUnit"
+    [displayInfoConfig]="{ disabledDisable: true }"
```

Comment:
Also breaking change as it changes template behaviour (ie. root button now being disabled by default). Need feature level flag solution for this like above comment.